highfidelity/hifi
# Tablet ui login logout

Test plan:
- Check login for existing user
- Try to log in non existing user. Make sure error showed
- Check Forgot username/password. Make sure browser within Tablet connected to HighFidelity passwords  restore facility
- check SignUp. Try to create user. Make sure user is created and logged in
- Press Cancel buttons. Make sure screen returns to previous or to main menu
- Press Existing user on Signup screen. Make sure it returns to previous screen

Known issue:
- Welcome screen not showed due to Tablet UI recreated on login
